cpp/arcticdb/version/symbol_list.hpp

```diff
@@ -50,6 +53,7 @@ class SymbolList {
     StreamId type_holder_;
     uint64_t max_delta_ = 0;
     std::shared_ptr<VersionMap> version_map_;
+    bool warned_expected_slowdown_ = false;
```

Comment:
any thread-safety concerns? Think this is the only mutable state

Reply:
Ah good question. Probably? I think this means it's best to remove!

Reply:
Reckon we probably do want it after all given what you said about,

```
If the user doesn't have write privileges this will repeat for every invocation. 
```

Reply:
Using std::atomic<bool>